api/services/registration_service.py

```diff
@@ -38,9 +39,14 @@ def create_registration(self, data):
         if registration_exists:
             return
 
+        user = models.User.query.filter_by(phone=data["from_number"]).first()

```

Comment:
Instead of quering in the service file. Lets create function inside User model and call it here? Also we should filter by using last 10 digit.

Reply:
updated